[Docs] Fix permanent project links to py.cafe (#642)
mckinsey/vizro: vizro-core/docs/pages/user-guides/run.md

## Py.Cafe

The easiest way to launch your dashboard is to edit the code live on [Py.Cafe](https://py.cafe/).

> Most of our examples in the docs have link below the code - [Edit live on Py.Cafe](https://py.cafe/vizro./vizro-iris-analysis-0) - which you can follow to experiment with the code of that example. This will lead you to an editor such as the one below, which displays the dashboard and the code side by side.

Reviewer: ```suggestion
Most of our examples have a link below the code — [Edit live on Py.Cafe](https://py.cafe/vizro./vizro-iris-analysis-0) — which you can follow to experiment with the code. This will lead you to an editor such as the one below, which displays the dashboard and the code side by side.
```

## Py.Cafe

The easiest way to launch your dashboard is to edit the code live on [Py.Cafe](https://py.cafe/).

Most of our examples have a link below the code — [Run and edit this code in Py.Cafe](https://py.cafe/vizro./vizro-iris-analysis-0) — which you can follow to experiment with the code. This will lead you to an editor such as the one below, which displays the dashboard and the code side by side.

<figure markdown="span">

>   ![Image title](../../assets/user_guides/run/pycafe_editor.png)

Reviewer: ```suggestion
  ![Py.Cafe editor](../../assets/user_guides/run/pycafe_editor.png)
```

> </div>

Reviewer 2: Is this supposed to be interactive so e.g. do we want people to be able to edit code from here? If yes, I think it's too blurry and zoomed out for any useful interaction. 

I would replace it with a high-resolution GIF instead if you just wanted to demo the editor mode?

Author: Good point, I replaced it with a screenshot in the end. I think that works best!
